libsource/src/main/java/com/virtusize/libsource/data/parsers/UserDataJsonParser.kt

```diff
@@ -18,9 +18,9 @@ internal class UserDataJsonParser: VirtusizeJsonParser {
 
     companion object {
```

Comment:
Those can be safely discarded, they are part of the legacy Purchase history flow and are not used in our modern products

Reply:
I removed the UserData class and only parsed should_see_ph_tooltip in the Data class. 
See the commit 27f90a3